.github/workflows/ci.yml

```diff
@@ -0,0 +1,334 @@
+name: 'CI'
+
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+  workflow_dispatch:
+  schedule:
+    - cron: "0 0 * * *"
+
+env:
+  APPLICATION_ENV: ci.pgsql
+  SPRYKER_TESTING_ENABLED: 1
+  APPLICATION_STORE: DE
+  PROJECT: suite
+  POSTGRES_PORT: 5433
+  POSTGRES_PASSWORD: postgres
```

Comment:
Why do we keep env variables here?

Reply:
Variables are used below for configuration db.